app/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -9,6 +9,14 @@ import PageTitle from 'components/PageTitle';
 class ErrorBoundary extends Component {
   @observable error: boolean = false;
 
+  componentWillReceiveProps(nextProps: Object) {
```

Comment:
This needed to be added as moving between documents doesn't remove the error. Location prop is optional for the component